src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -228,14 +228,16 @@ public void canCheckOutUsingDueDateLimitedRollingLoanPolicy() {
 
     final IndividualResource loanPolicyResource = loanPoliciesFixture.create(
       dueDateLimitedPolicy);
+    final IndividualResource overdueFinePolicyResource = overdueFinePoliciesFixture.facultyStandard();
```

Comment:
What extra information does the `resource` suffix provide for this variable name?

```suggestion
    final IndividualResource overdueFinePolicy = overdueFinePoliciesFixture.facultyStandard();
```

Reply:
the `resource` suffix has been added to be consistent with `loanPolicyResource` which is used above `overdueFinePolicy `.  There are two options: to leave the `resource `suffix for this variable name or remove it from `loanPolicyResource` as well.

Reply:
Ah, ok, thank you for that context. 

Personally, I would remove the `resource` context from all of these variable names.

Reply:
Fixed